### DISCUSSION TITLE
Will v1 continue to receive updates?

### DISCUSSION THREAD

**Discussion by CarbonPool** [tauri-apps/tauri#11298] [Q&A]
When I read the v2 documentation, I found that most of the original APIs were split into independent plugins, which required more configuration and dependencies, and more API changes, making migration difficult. Why did the team think so?

**Comment by FabianLars** [comment]
> Will v1 continue to receive updates?

Yes, but the team will mostly only work on bug fixes. New features need community PRs.

> When I read the v2 documentation, I found that most of the original APIs were split into independent plugins, which required more configuration and dependencies, and more API changes, making migration difficult. Why did the team think so?

1) To make it easier to work on the apis and update them independently from tauri
2) To make it much easier for others (non team members) to contribute meaningful changes.
